great-idea-website/css/index.css

```diff
@@ -49,18 +130,116 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+html,
+body {
+	height: 100%;
+	font-family: 'Titillium Web', sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+	font-family: 'Bangers', cursive;
+	letter-spacing: 1px;
+	margin-bottom: 15px;
 }
 
-h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
-    letter-spacing: 1px;
-    margin-bottom: 15px;
+/* Your code starts here! */
+
+.header1 {
+	margin-bottom: 20px;
+}
+
+.navigation {
+	display: inline-block;
+	font-size: 8;
+	padding-bottom: 20px;
+	margin-right: 20px;
+}
+
+.navigation a {
+	color: lightgray;
+	text-decoration: transparent;
+	margin: 20px;
+}
+.header .logo {
+	position: absolute;
+	right: 0px;
+	margin: auto;
 }
+.top-section {
+	height: 400px;
+	position: relative;
 
-/* Your code starts here! */
\ No newline at end of file
+	margin: auto;
+}
+.top-section h1 {
+	font-size: 60px;
+	padding-left: 100px;
+	padding-top: 80px;
+	display: inline-block;
+}
+
+.top-section button {
+	padding: 7px 40px;
+	border-radius: 1px;
+	background-color: white;
+	font-size: 12px;
+}
+
+.top-section img {
+	display: inline-block;
+	box-align: right;
+	position: absolute;
```

Comment:
Because of position absolute the top section image will stick to the right side of the browser's window. It will be "sticked" there so when someone browses the page with a higher resolution the whole website will go crazy. Better way is to contain it along with other element in a div and try to position it with display inline block and widths.